app/views/taxCodeChange/TaxCodeComparisonView.scala.html

```diff
@@ -14,37 +14,41 @@
  * limitations under the License.
  *@
 
-@import includes.ptaHeader
-@import includes.details_summary
 @import controllers.routes
-@import uk.gov.hmrc.tai.model.domain.TaxCodeRecord
-@import uk.gov.hmrc.tai.viewModels.taxCodeChange.{TaxCodeChangeViewModel, TaxCodePairs}
+@import includes.ptaHeader
 @import uk.gov.hmrc.play.views.formatting.Dates
-@import uk.gov.hmrc.tai.viewModels.DescriptionListViewModel
-@import org.joda.time.LocalDate
-@import uk.gov.hmrc.urls.Link
-@import uk.gov.hmrc.tai.viewModels.GoogleAnalyticsSettings
 @import uk.gov.hmrc.tai.config.ApplicationConfig
+@import uk.gov.hmrc.tai.model.domain.TaxCodeRecord
 @import uk.gov.hmrc.tai.util.HtmlFormatter
+@import uk.gov.hmrc.tai.viewModels.GoogleAnalyticsSettings
+@import uk.gov.hmrc.tai.viewModels.taxCodeChange.{TaxCodeChangeViewModel, TaxCodePairs}
+@import uk.gov.hmrc.urls.Link
+@import views.html.taxCodeChange.FormattedTaxCodeForComparison
+
+@this(mainTemplate: MainTemplate)
 
-@this(main: main)
 
-@(taxCodeChangeViewModel: TaxCodeChangeViewModel, appConfig: ApplicationConfig)(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+@(taxCodeChangeViewModel: TaxCodeChangeViewModel,appConfig: ApplicationConfig)(
+    implicit request: Request[_],
+    user: controllers.auth.AuthedUser,
+    messages: Messages,
+    templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,
+    ec:  scala.concurrent.ExecutionContext)
 
 @gaEventActionString = @{"Your tax code change"}
-@gaEventLabelString = @{Messages("taxCode.change.yourTaxCodeChanged.checkYourTaxButton")}
+@gaEventLabelString = @{messages("taxCode.change.yourTaxCodeChanged.checkYourTaxButton")}
```

Comment:
@Sachin300793 why has messages changes throughout this file randomly? They should all be lower case messages here as this is the one we are implicitly passing in

Reply:
Made changes to use lower case messages , done